Windows 8 x64 install error

d:\project>npm install pg
npm http GET https://registry.npmjs.org/pg
npm http 304 https://registry.npmjs.org/pg
npm http GET https://registry.npmjs.org/generic-pool/2.0.3
npm http GET https://registry.npmjs.org/buffer-writer/1.0.0
npm http GET https://registry.npmjs.org/nan
npm http 304 https://registry.npmjs.org/generic-pool/2.0.3
npm http 304 https://registry.npmjs.org/buffer-writer/1.0.0
npm http 304 https://registry.npmjs.org/nan

> pg@2.8.4 install d:\project\node_modules\pg
> node-gyp rebuild || (exit 0)

d:\project\node_modules\pg>node "C:\Program Files\nodejs\node_modules\npm\bin\node-gyp-bin\....\node_modules\node-gyp\bin\node-gyp.js" rebuild
Building the projects in this solution one at a time. To enable parallel build, please add the "/m" switch.
  binding.cc
..\src\binding.cc(3): fatal error C1083: Cannot open include file: 'nan.h': No such file or directory [d:\project\node_modules\pg\build\binding.vcxproj]
gyp ERR! build error
gyp ERR! stack Error: `msbuild` failed with exit code: 1
gyp ERR! stack     at ChildProcess.onExit (C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\lib\build.js:267:23)
gyp ERR! stack     at ChildProcess.EventEmitter.emit (events.js:98:17)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (child_process.js:789:12)
gyp ERR! System Windows_NT 6.2.9200
gyp ERR! command "node" "C:\Program Files\nodejs\node_modules\npm\node_modules\node-gyp\bin\node-gyp.js" "rebuild"
gyp ERR! cwd d:\project\node_modules\pg
gyp ERR! node -v v0.10.23
gyp ERR! node-gyp -v v0.12.1
gyp ERR! not ok
pg@2.8.4 node_modules\pg
├── nan@0.6.0
├── buffer-writer@1.0.0
└── generic-pool@2.0.3

d:\project>_

--- a/binding.gyp
+++ b/binding.gyp
@@ -5,7 +5,7 @@
       'sources': ['src/binding.cc'],
       'include_dirs': [
         '<!@(pg_config --includedir)',
-        '<!(node -e \'require("nan")\')'
+        '<!(node -e "require(\'nan\')")'
       ],
       'conditions' : [
         ['OS=="win"', {

